Scripts/build.swift

```diff
@@ -95,22 +127,40 @@ enum Platform: String, CustomStringConvertible {
         case .iOS_13,
              .iOS_14,
              .iOS_15,
+             .iOS_16,
+             .iOS_17,
              .tvOS_13,
              .tvOS_14,
              .tvOS_15,
+             .tvOS_16,
+             .tvOS_17,
              .macOS_10_15,
              .macOS_11,
-             .macOS_12:
+             .macOS_12,
+             .macOS_13,
+             .macOS_14:
             return true
 
         case .watchOS_6,
              .watchOS_7,
-             .watchOS_8:
+             .watchOS_8,
+             .watchOS_9,
+             .watchOS_10:
             // watchOS does not support unit testing (yet?).
             return false
         }
     }
 
+    var requiresUpdatedSPMIntegration: Bool {
```

Comment:
Would love a comment here. Otherwise maybe we could rename to `supportsGeneratedXcodeproj` and invert the check? As-is it's not clear what is "updated" about the integration.

Reply:
Added.